app/src/main/res/values/themes.xml

```diff
@@ -13,4 +13,17 @@
         <item name="android:statusBarColor" tools:targetApi="l">?attr/colorPrimary</item>
         <!-- Customize your theme here. -->
     </style>
+    <style name="Theme.AlgoPhantoms1" parent="Theme.MaterialComponents.DayNight.NoActionBar">
```

Comment:
consider naming themes based on what it holds, for there `Theme.AlgoPahantoms.NoActionBar` makes more sense

Reply:
yeah . Thanks for pointing out. let me commit required changes and let you know.